src/main/java/frc/robot/robotcontainers/RobotContainer.java

```diff
@@ -15,11 +17,12 @@
  */
 public abstract class RobotContainer {
     public Command getAutonomousCommand() {
-        return new InstantCommand();
+        return new PathPlannerAuto("example auto");
```

Comment:
This is the RobotContainer abstract class. This change to getAutonomousCommand is propagated to all RobotContainers that don't override this default implementation. Which since your robot container overrides it, this change shouldn't do anything, but it should be put back to InstantCommand so that someone with beelte doesn't accidently run an auto or something.

Also you should switch back to using Poseidon container with Poseidon. Robot id 0 will be poseidon for the foreseeable future (which is something I need to talk about in person)